### PR TITLE
copied in latest changes from master branch

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#6581

*Description of changes:*
- copied in Hotjar from master branch
- updated app version to 2.0.4
- copied in Webchat changes from master branch
- copied in nginx config (disable index.html caching)
- updated webchat status error handling

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: public/config/configuration.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
     "AUTH_TOKEN_URL": "https://dev.oidc.gov.bc.ca/auth/realms/fcf0kpqr/protocol/openid-connect/token",
     "QUICK_SEARCH_PUBLIC_ID": "",
     "QUICK_SEARCH_PUBLIC_SECRET": "",
-    "ENTITY_SELECTOR_URL": "https://entity-selection-dev.pathfinder.gov.bc.ca/"
+    "HOTJAR_ID": "2284327",
```
Comment (severinbeauvais) [inline public/config/configuration.json:18]: This is the Hotjar ID for local dev. There area other ones for Dev, Test and Prod.

File: src/components/common/chat-popup.vue [hunk 2]
```diff
@@ -43,52 +44,61 @@
...
-    MainContainer
+  chatStatus = null
+  chatError = false
+
+  mounted (): void {
+    axios
+      .get(window['webChatStatusUrl'])
+      .then(response => (this.chatStatus = response.data.status))
+      .catch(error => {
+        console.log(error)
```
Comment (cameron-freshworks) [inline src/components/common/chat-popup.vue:67]: Do we want this to be `console.error`? 
Can't recall whats best for Sentry..

**Review by cameron-freshworks — APPROVED** [review]
LGTM, merge away after reading the comment!

File: src/components/common/chat-popup.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/common/chat-popup.vue:67]: Sentry captures errors, yes.
I'll look at this again. At a minimum, it's missing a lint bypass.

Comment (cameron-freshworks) [reply src/components/common/chat-popup.vue:67]: No biggie though, merge away if you're good with it!

Comment (severinbeauvais) [reply src/components/common/chat-popup.vue]: Fixed.
(It did work before -- the button is disabled -- but Sentry would not have captured the error.)
Yes, I think this error should go to Sentry.

**Pull request merged**